build-parsers.sh

```diff
@@ -28,5 +28,7 @@ fi
 
 #Generate the haxe and hxml parsers
 java -jar grammar-kit.jar gen ./grammar/haxe.bnf
-java -jar grammar-kit.jar gen ./src/com/intellij/plugins/haxe/hxml/hxml.bnf
+#java -jar grammar-kit.jar gen ./src/common/com/intellij/plugins/haxe/hxml/hxml.bnf
```

Comment:
The path `./src/com/` does not exists and this command silently fails. Yes, we can fix the path, but the Grammar Kit 1.2.0 will generate code with a lot of incompatibilities for current project source. Every time I run this script, I had to revert all the hxml parsers.
Despite this should I uncomment this command?